### PR TITLE
BVSparse: keep 'last found' for fast fromIndex discovery

### PR DESCRIPTION
**Pull request by obastemur**
Helps the number of tests not to timeout on CI machines.

### CODE REVIEW

File: lib/Common/DataStructures/SparseBitVector.h [hunk 1]
```diff
@@ -486,6 +508,7 @@ template <class TAllocator>
 BVSparseNode<TAllocator> *
 BVSparse<TAllocator>::DeleteNode(BVSparseNode *node, bool bResetLastUsed)
 {
+    this->tail = nullptr;
```
Comment (Penguinwizzard) [inline lib/Common/DataStructures/SparseBitVector.h]: Do we necessarily only delete the last node? If so, shouldn't we update the tail to be the new last node?

Comment (obastemur) [reply lib/Common/DataStructures/SparseBitVector.h]: updated the variable name.

Comment (Penguinwizzard) [reply lib/Common/DataStructures/SparseBitVector.h]: Ah, that clears things up a lot, thanks!

**Comment by obastemur** [comment]
@Penguinwizzard Thanks for the review!

**Pull request merged**